### PR TITLE
feature/81

### PR DESCRIPTION
**Pull request by caymanh**
1. installed react-modal to simulate popup card since the component has not been created
2. added an "Email" button on /driver-search as a placeholder for the email icon which has not been added to the page
3. imported useState to /drive-search for handling the display and hiding of the popup
4. added functionality to display popup when users click on the "Email" button
5. added functionality to close the popup when users click on either the "Close" button in the card or outside the Card area

<|<- META INFO (DO NOT MODIFY CONTENTS MANUALLY) ->|>
*BLOCKED BY*
No Issues

*BLOCKING*
No Issues

<|<- META INFO END ->|>

### CODE REVIEW

File: web-client/package.json [hunk 1]
```diff
@@ -22,12 +22,14 @@
...
     "@netlify/plugin-nextjs": "^1.1.3",
     "@reduxjs/toolkit": "^1.5.0",
+    "@types/react-modal": "^3.12.0",
     "@types/react-redux": "^7.1.15",
     "fishery": "^1.2.0",
     "next": "^10.0.9",
     "react": "17.0.1",
     "react-dom": "17.0.1",
     "react-hook-form": "^6.14.2",
+    "react-modal": "^3.12.1",
```
Comment (ynynl) [inline web-client/package.json:32]: Hey @caymanh, I think you can use Material UI for popup https://material-ui.com/components/dialogs/

Comment (caymanh) [reply web-client/package.json]: hi @ynynl, thanks for your feedback! I have uninstalled the react-modal package and used Material UI to create a placeholder popup component instead. let me know if you have any other comments.

File: web-client/src/shared/components/drivers-search/List/DriverCard/index.tsx [hunk 2]
```diff
@@ -26,7 +30,11 @@ export const DriverCard = ({ driverInfo }: DriverCardProps): JSX.Element => {
             {driverInfo.name}
           </span>
         </Typography>
-
+        <button onClick = {() => setOpenPopup(true)}>Email</button>
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-search/List/DriverCard/index.tsx:33]: Do you think you could use material ui components where possible, like for the button

File: web-client/src/shared/components/drivers-search/Popup/index.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+interface PopupProps {
+    openPopup: boolean;
+    setOpenPopup: (arg0: boolean) => void;
+}
+
+export const Popup = ({ openPopup, setOpenPopup }: PopupProps): JSX.Element => {
+    return (
+        <Dialog open={openPopup}>
+            <DialogTitle>
+                <div>Placeholder for Title</div>
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-search/Popup/index.tsx:13]: Looks like the details of the popup are a little off. We need all the information included in the design spec for this ticket.

**Review by l2silver — CHANGES_REQUESTED** [review]
So, I've added a couple of comments. Looks like we need to make sure the popup content matches the designs. And I need you to change the name Popup to something more specific to this particular component. Something like MessagingPopup , and change it everywhere that is appropriate. Thanks @caymanh  great work!

File: web-client/src/shared/components/drivers-search/List/DriverCard/index.tsx (hunk 2 above)
Comment (caymanh) [reply web-client/src/shared/components/drivers-search/List/DriverCard/index.tsx:33]: @l2silver thanks for your feedback! yes, I can replace the button with material UI component

File: web-client/src/shared/components/drivers-search/Popup/index.tsx (hunk 3 above)
Comment (caymanh) [reply web-client/src/shared/components/drivers-search/Popup/index.tsx:13]: @l2silver noted. I will rename the Popup component to MessagingPopup. Regarding your comment on the details of the popup - the reason why I did not fully build out the popup according to the design spec was that I thought that the popup card itself is a separate ticket (#83) and hence, I merely used a placeholder for now while waiting for that ticket to be assigned and completed before I replaced the current placeholder with the actual card. however, I'm happy to build out the actual card as per ticket #83 if that works better for you.

Comment (l2silver) [reply web-client/src/shared/components/drivers-search/Popup/index.tsx]: Omg @caymanh you're absolutely right! Sorry about that. The placeholder is perfect!

**Pull request merged**